shell/platform/android/external_view_embedder/external_view_embedder_unittests.cc

```diff
@@ -367,8 +369,9 @@ TEST(AndroidExternalViewEmbedder, SubmitFrame__RecycleSurfaces) {
     // frame.
     EXPECT_CALL(*jni_mock, FlutterViewCreateOverlaySurface()).Times(0);
     // The JNI call to display the Android view.
-    EXPECT_CALL(*jni_mock,
-                FlutterViewOnDisplayPlatformView(0, 100, 100, 300, 300));
+    MutatorsStack mutators_stack;
```

Comment:
we can probably add a unit test to ensure that the Mutator stack is passed correctly

Reply:
yeah, this failed anyways, i fixed it so now it makes sure the mutator stack is passed in correctly.